octobot.py

```diff
@@ -195,22 +210,23 @@ def _create_symbol_threads_managers(self, exchange, symbol_evaluator):
                                                                                 exchange,
                                                                                 symbol_evaluator.get_symbol(),
                                                                                 self.relevant_evaluators)
-        symbol_time_frame_updater_thread = SymbolTimeFramesDataUpdaterThread()
```

Comment:
Can we also remove the `SymbolTimeFramesDataUpdaterThread` file ?

Reply:
yes, when everything is finished (used as a reference for now)